shared/src/business/entities/DocketEntry.ts

```diff
@@ -143,7 +143,25 @@ export class DocketEntry extends JoiValidationEntity {
   public privatePractitioners?: any[];
   public servedParties?: any[];
   public signedAt?: string;
-  public draftOrderState?: object;
+  public draftOrderState?: {
```

Comment:
I love that you are typing this! Can you give me some insight into how you found these properties? If I do a search for `draftOrderState.`, I also get `docketEntryDescription`, `editorDelta`, and `documentContents` and I don't get `generatedDocumentTitle`. (I haven't looked through them all, this is just one I can't find). I'm not sure if this is because I am looking at the wrong things or not!

Reply:
No special insights on offer: I think you're just more thorough than I am! Thank you for pointing this out.